features/step_definitions/networking.rb

```diff
@@ -308,10 +308,12 @@
 Given /^the AllowNamespaceAndPod policy is applied to the "(.+?)" namespace$/ do | project_name |
   ensure_admin_tagged
 
-  @result = admin.cli_exec(:create, n: project_name , f: "#{BushSlicer::HOME}/testdata/networking/networkpolicy/allow-ns-and-pod.yaml")
-  unless @result[:success]
-    raise "Failed to apply the default deny policy to specified namespace."
-  end
+  step %Q{I obtain test data file "networking/networkpolicy/allow-ns-and-pod.yaml"}
+  step %Q/I run the :create admin command with:/,table(%{
+    | f | allow-ns-and-pod.yaml |
+    | n | #{project_name} |
```

Comment:
@asood-rh table alignment.  You could've also just do` @result = admin.cli_exec(:create, n: project_name , f: "allow-ns-and-pod.yaml")`

Reply:
Fixed alignment.